src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -1,451 +1,242 @@
-          <AccordionSummary
-            expandIcon={<ExpandMoreIcon />}
-            aria-controls="panel1a-content"
-            id="panel1a-header"
+  }
+  return loading ? (
+    <GordonLoader />
+  ) : (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
+      {
+        <>
+          <Accordion
+            TransitionProps={{ unmountOnExit: true }}
+            onChange={handleIsExpanded}
+            defaultExpanded={props.myProf}
           >
-            <Typography>{panelTitle} the Schedule</Typography>
-          </AccordionSummary>
-          <AccordionDetails>
-            <Grid container direction="row" justifyContent="center">
-              {this.props.isOnline && (
-                <Grid container direction="row" item xs={12} lg={10}>
-                  <Grid
-                    container
-                    item
-                    xs={12}
-                    lg={8}
-                    alignItems="center"
-                    justifyContent="flex-start"
-                  >
-                    <Markup content={replaced} />
-                  </Grid>
-
-                  <Grid
-                    container
-                    direction="column"
-                    item
-                    xs={12}
-                    lg={4}
-                    alignItems="flex-end"
-                    justifyContent="flex-end"
-                  >
-                    {privacyButton}
-                  </Grid>
-
-                  <Grid item xs={6} lg={2}>
-                    {editDescriptionButton}
-                  </Grid>
-
-                  <Grid item xs={6} lg={2}>
-                    {removeOfficeHourButton}
-                  </Grid>
-
-                  <Grid
-                    container
-                    direction="column"
-                    item
-                    xs={12}
-                    lg={8}
-                    alignItems="flex-end"
-                    justifyContent="flex-end"
-                  >
-                    {lastUpdate}
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon className={styles.expandIcon} />}
+              aria-controls="panel1a-content"
+              id="panel1a-header"
+              className={styles.header}
+            >
+              <CardHeader
+                className={styles.accordionHeader}
+                title={isExpanded ? 'Hide Schedule' : 'Show Schedule'}
+              />
+            </AccordionSummary>
+            <AccordionDetails>
+              <Grid container direction="row" justifyContent="center" align="left" spacing={4}>
+                {props.isOnline && (
+                  <Grid container direction="row" item xs={12} lg={12} spacing={2}>
+                    <Grid item lg={1}></Grid>
+                    <Grid item xs={4} lg={1} align="left" className={styles.officeHourText}>
+                      <Markup content="Office Hours: " />
```

Comment:
(We discussed this in person, but recording here.) The title and the edit pop-up should agree on what this is called. Let's use "Public Office Hours Note", as in the pop-up, at least while there is also another Office Hours link.